web/pandas/getting_started.md

```diff
@@ -1,5 +1,29 @@
 # Getting started
 
+## Try it in your browser
+
+You can try `pandas` in your browser with the following interactive shell
+without installing anything on your computer.
+
```

Comment:
Maybe worth adding a note that the shell takes a bit of time to load/respond the first time, to manage expectations?